test/e2e/config/e2e_conf.yaml

```diff
@@ -129,6 +132,8 @@ variables:
   # Pin Calico version
   CALICO_PATCH_RELEASE: "v3.21.0"
   CALICO_MINOR_RELEASE: "v3.21"
+  # Pin CertManager for upgrade tests
+  CERT_MANAGER_RELEASE: v1.5.3
```

Comment:
@mboukhalfa what is the reason behind pinning it to this specific version? due to v1a5? please keep in mind https://github.com/metal3-io/cluster-api-provider-metal3/pull/755 is bumping it to 1.9.1

Reply:
yes this is the version used by CAPI `v0.4.8` I am not sure if we uplifted to a newer version what will happen after `clusterctl init ` anyway I prefer to merge this PR then #755 will be rebased and can use the version the more adequate

Reply:
755 bumps it in Tiltfile, so this should be fine.